bin/accessibility-automation/helper.js

```diff
@@ -0,0 +1,193 @@
+const logger = require("../helpers/logger").winstonLogger;
+const { API_URL } = require('./constants');
+const utils = require('../helpers/utils');
+const fs = require('fs');
+const path = require('path');
+const request = require('request');
+const os = require('os');
+const glob = require('glob');
+const helper = require('../helpers/helper');
+
+exports.checkAccessibilityPlatform = (user_config) => {
+  let accessibility = false;
+  try {
+    user_config.browsers.forEach(browser => {
+      if (browser.accessibility) {
+        accessibility = true;
+        return true;
```

Comment:
We can omit this return, it has no effect over here. If the expected behaviour is to break out of this loop then better use a normal `for` loop

Reply:
done